### ISSUE TITLE
Twitter Tracking

### ISSUE THREAD

**Issue by ccbrown** [ccbrown/gggtracker#12]
It's 95% redundant, but it seems like every now and then something gets posted exclusively to Twitter. Example: [Tweet](https://twitter.com/pathofexile/status/887910063293079554) and [relevant conversation](https://www.reddit.com/r/pathofexile/comments/6oec9o/confirmation_that_the_new_mtx_system_will_be/dkgvqlf/).

I'm open to tracking Twitter under one condition: It shouldn't pollute the activity feed with redundant announcements. If a Tweet just links to a forum post, either the tweet should be ignored, or the two of them should be coalesced into a single feed item. Example: If [this tweet](https://twitter.com/pathofexile/status/887906946837233664) is posted after [this forum post](https://www.pathofexile.com/forum/view-post/14622421), the tweet should not appear as a separate feed item.

I'm open to pull requests if someone wants to give this a shot.

**Comment by Tritiumnzlol** [comment]
Yeah this could be a real bloat on the comment tracker. Whoever ends up implememting this could  do any of the following things fewings to mitigate such bloat:

* Have tweet tracking default to being off
* Just have the Twitter users display picture and display the tweet when it is hovered over in a tooltip
* Use the links contained in tweets as a foreign key of news posts to link them to "normal" news posts, allowing a Twitter icon to be displayed in the same row

As a side question I'm not sure where you stand on a permissions basis, or wish to include with employees personal Twitter accounts i.e. bex's personal twitter does contain a fair amount of Poe/GGG related stuff https://twitter.com/bexchangewords

**Comment by TimboZero** [comment]
This could be added as a feature with toggle switches to enable/disable Twitter, GGG Forums and Reddit so readers can select which are or aren’t included.
